README.md

```diff
@@ -209,6 +209,24 @@ select * from {{ ref('users') }}
 {% endif %}
 ```
 
+##### `merge`
+
+Through the `merge` incremental strategy, dbt-trino constructs `MERGE` statement which `INSERT` new records and `UPDATE` existing ones based on unique key specified by `unique_key`. 
```

Comment:
add a link for `MERGE` https://trino.io/docs/current/sql/merge.html

Do emphasize the fact that `MERGE` is not available on all the connectors.